lib/client-assets.php

```diff
@@ -419,7 +419,7 @@ function gutenberg_register_packages_styles( $styles ) {
 		$styles,
 		'wp-edit-site',
 		gutenberg_url( 'build/edit-site/style.css' ),
-		array( 'wp-components', 'wp-block-editor', 'wp-editor', 'wp-block-library-editor', 'common', 'forms', 'wp-commands', 'wp-preferences' ),
+		array( 'wp-components', 'wp-block-editor', 'wp-editor', 'wp-edit-blocks', 'wp-block-library-editor', 'common', 'forms', 'wp-commands', 'wp-preferences' ),
```

Comment:
@ellatrix, @youknowriad do you remember if `wp-edit-blocks` was omitted intentionally? It's a dependency for all other `edit-*` apps.

Reply:
I guess because it's about the "content", so it needs to be loaded in the iframe, not in the page itself. 

I think the issue here is that the warning shouldn't be in "content", it should be in "styles" just like the other components (like placeholder...)

Reply:
IIRC, the site editor was always iframed, so that can explain the dependency difference.

> I think the issue here is that the warning shouldn't be in "content", it should be in "styles" just like the other components (like placeholder...)

Any component that's supposed to render in iframe canvas uses content styles, including MediaPlaceholder.

I think I'll duplicate styles for ErrorBounday and might just stop using the `Warning` component from the block editor. Wanted to avoid that, but pulling the whole stylesheet just for one component doesn't make sense.

I also realized we can use the error boundary component from the `editor` package now that `edit-site` depends on it.

https://github.com/WordPress/gutenberg/blob/b05a01f155ffd7780f56be383f9684c2c6877fac/packages/block-editor/src/content.scss#L1-L14